src/i18n.js

```diff
@@ -181,6 +195,7 @@ i18n
               text: 'Are you sure you want to delete your project \'{{name}}\'?'
             },
             empty: 'No projects created yet',
+            label: 'Action menu',
```

Comment:
Same as the above, maybe something like `Project actions menu` or `Open project menu`?